widget/embedded/src/pages/WalletsPage.tsx

```diff
@@ -146,13 +149,21 @@ export function WalletsPage({
             sortedWallets.map((wallet, index) => {
```

Comment:
Use i18n for line 142 (for this text: Choose a wallet to connect. )

Reply:
fixed.